### DISCUSSION TITLE
How to know all the built-in helper functions

### DISCUSSION THREAD

**Discussion by nadeemkhedr** [johnlindquist/kit#416] [Guide]
I do know there are so many nice utils, is there a doc listing them?

**Comment by johnlindquist** [comment]
I'm working on docs this week.

You can browse the globals in these two dirs (on the "beta" branch, "main" is waaaaaay out-of-date):

lib utils:
https://github.com/johnlindquist/kit/tree/beta/src/lib

package utils:
https://github.com/johnlindquist/kit/tree/beta/src/api/packages

**Reply by nadeemkhedr** [reply]
Awesome! Thanks

**Reply by nadeemkhedr** [reply]
What is the easiest way to switch to beta and pull the updates?

**Reply by johnlindquist** [reply]
If you've installed Kit.app, you're on the beta branch.

I doubt anyone is on main. We'll do a main release once the docs are finalized. 

The big impact is the auto-updater uses "release channles", so it will update from beta->main and then most users only update on the main branch (unless they explicitly switch back to beta).

**Comment by patrikzudel** [comment]
Any update on this?

Honestly I assume **this** is the biggest hurdle in actual adoption of this program, it is beautiful and elegant in both UI and UX, but there simply isn't sufficient documentation for developing scripts. 

Having **all** of the built-in helper functions in one place, each having a simple description and a single usage example would literally be a game changer. (Including listing their OS compatibility)

**Reply by johnlindquist** [reply]
I really appreciate the kind words 💛

I almost have the Script Kit 2.0 release candidates ready, I'm just polishing up a couple Windows-specific issues. Hoping tomorrow/Monday.

That should open up my time to finally start documenting features/APIs so everyone can find out what they're missing on :)
